docs/getting-started/cli-guide.md

```diff
@@ -655,3 +658,264 @@ sudo bpfman load image --image-url quay.io/bpfman-bytecode/xdp_pass:latest --pul
+          Example: --bc-amd64-el ./examples/go-xdp-counter/bpf_x86_bpfel.o
+
+      --bc-arm-el <BC_ARM_EL>
+          Optional: bytecode file to use for building the image assuming arm architecture.
+          Example: --bc-arm-el ./examples/go-xdp-counter/bpf_arm_bpfel.o
+
+      --bc-arm64-el <BC_ARM64_EL>
+          Optional: bytecode file to use for building the image assuming arm64 architecture.
+          Example: --bc-arm64-el ./examples/go-xdp-counter/bpf_arm64_bpfel.o
+
+      --bc-loong64-el <BC_LOONG64_EL>
+          Optional: bytecode file to use for building the image assuming loong64 architecture.
+          Example: --bc-loong64-el ./examples/go-xdp-counter/bpf_loong64_bpfel.o
+
+      --bc-mips-eb <BC_MIPS_EB>
+          Optional: bytecode file to use for building the image assuming mips architecture.
+          Example: --bc-mips-eb ./examples/go-xdp-counter/bpf_mips_bpfeb.o
+
+      --bc-mipsle-el <BC_MIPSLE_EL>
+          Optional: bytecode file to use for building the image assuming mipsle architecture.
+          Example: --bc-mipsle-el ./examples/go-xdp-counter/bpf_mipsle_bpfel.o
+
+      --bc-mips64-eb <BC_MIPS64_EB>
+          Optional: bytecode file to use for building the image assuming mips64 architecture.
+          Example: --bc-mips64-eb ./examples/go-xdp-counter/bpf_mips64_bpfeb.o
+
+      --bc-mips64le-el <BC_MIPS64LE_EL>
+          Optional: bytecode file to use for building the image assuming mips64le architecture.
+          Example: --bc-mips64le-el ./examples/go-xdp-counter/bpf_mips64le_bpfel.o
+
+      --bc-ppc64-eb <BC_PPC64_EB>
+          Optional: bytecode file to use for building the image assuming ppc64 architecture.
+          Example: --bc-ppc64-eb ./examples/go-xdp-counter/bpf_ppc64_bpfeb.o
+
+      --bc-ppc64le-el <BC_PPC64LE_EL>
+          Optional: bytecode file to use for building the image assuming ppc64le architecture.
+          Example: --bc-ppc64le-el ./examples/go-xdp-counter/bpf_ppc64le_bpfel.o
+
+      --bc-riscv64-el <BC_RISCV64_EL>
+          Optional: bytecode file to use for building the image assuming riscv64 architecture.
+          Example: --bc-riscv64-el ./examples/go-xdp-counter/bpf_riscv64_bpfel.o
+
+      --bc-s390x-eb <BC_S390X_EB>
+          Optional: bytecode file to use for building the image assuming s390x architecture.
+          Example: --bc-s390x-eb ./examples/go-xdp-counter/bpf_s390x_bpfeb.o
+
+  -h, --help
+          Print help (see a summary with '-h')
+```
+
+Below are some different examples of building images.
+Note that `sudo` is not required.
+This command also pushed the image to a registry, so user must already be logged into the registry.
+
+Example of single bytecode image:
+
+```console
+bpfman image build -f Containerfile.bytecode -t quay.io/$QUAY_USER/go-xdp-counter:test -b ./examples/go-xdp-counter/bpf_x86_bpfel.o
+```
+
+Example of directory with Cilium generated bytecode objects:
+
+```console
```

Comment:
Do we need the following note somewhere around here?

To build images for multiple architectures on a local system, docker may need additional configuration settings to allow for caching of non-native images. See https://docs.docker.com/build/building/multi-platform/ for more details.